example/grpc-example/README.md

```diff
@@ -1,39 +1,94 @@
-# GRPC sample
-
-## Introduction
-
+# Overview
 This sample is based on [gRPC Hello World](https://github.com/grpc/grpc-go/tree/master/examples).
-It demonstrates how convert existing gRPC application to communicate over a Ziti Network
+It demonstrates how convert existing gRPC application to communicate over an app embedded zero trust 
+OpenZiti Network
 
-## Instructions
+This example demonstrates:
+* Binding a service and listening for service calls
+* Dialing a service and triggering service calls
 
-### Build the code
+## Requirements
+* go 1.19 or later
+* an OpenZiti network. If you do not have one, you can use one of the [quickstarts](https://openziti.github.io/ziti/quickstarts/quickstart-overview.html) to set one up.
+* OpenZiti CLI to create services and identities on the OpenZiti Network
 
-```console
-$ go get github.com/openziti/sdk-golang/example/grpc-example/grpc-client
-$ go get github.com/openziti/sdk-golang/example/grpc-example/grpc-server
+## Build the examples
+```shell
+cd <repo-root-dir>/sdk-golang/example/grpc-example/
+cd grpc-server
+go mod tidy && go build -o grpc-server main.go
```

Comment:
is it possible to cd to 'this' folder and then just run:
```
go build -o grpc-server grpc-server/main.go
go build -o grpc-client grpc-client/main.go
```
then you can refer to the path as just `grpc-server/main.exe` and `grpc-client/main.exe` below without the changing directories

Reply:
I had tried this as I used it on a previous example. However, because it has a mod file I couldn't get this to work. Here is an example of the output when you try to run build with -o

```
grpc-server/main.go:8:2: missing go.sum entry for module providing package google.golang.org/grpc; to add:
	go mod download google.golang.org/grpc
grpc-server/main.go:9:2: missing go.sum entry for module providing package google.golang.org/grpc/examples/helloworld/helloworld; to add:
	go mod download google.golang.org/grpc
```

Any idea how to get around this? I tried also add `--modfile` and pointing to the modfile but that also doesn't seem to work.
```
$ go build -o grpc-server grpc-server/main.go --mo
dfile grpc-server/go.mod
named files must be .go files: --modfile
```